<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:  #C2272D;
}

QLineEdit{
	background-color: #fff;
	color: #000000;
	border-radius:  15px;
	padding-left : 11px;
}

QPushButton{
background-color: #fff;
text-transform: uppercase;
border-radius:  15px;
color: #000000;

}

QToolButton
{
    border-radius: 30px;
}

QMenu{
background-color:  gray;
color: #fff;
}

QMenuBar{
background-color:  #434343;
color: #fff;
}

QAction{
background-color: gray;
color: #fff;

}

QMenuBar::item:selected{
background-color: gray;
color: #fff;
}

QMenu::item:selected{
background-color:  #404040;
}

QMessageBox::QPushButton{
padding: 15px;
}

QLabel{
	color: #fff;
}

QMessageBox QLabel {
	color: #000000;
}

QInputDialog &gt; QLabel{
	color:#000000;
}

QInputDialog &gt; QPushButton{
	margin: 4px;
	padding: 5px;
}

QInputDialog &gt; QLineEdit{
	font-size: 13px;
	padding: 5px;
	border-radius: 0px;
}
/* ---------------------------*/
QDialog &gt; QLabel{
	color: #000000;
}

QDialog &gt; QPushButton{
	padding: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #fff;
}</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="adminBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Employee Actions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seeAllEmployeesButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>See All Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addEmployeeButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Employee</string>
     </property>
    </widget>
    <widget class="QPushButton" name="revenueReportButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Revenue Report</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editEmployeeButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>EDIT EMPLOYEE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insuranceDistributionReportButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Insurance Distribution Report</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Reports</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="loadOptionsLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cannot Load Options</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="deskBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Desk Actions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newCustomerButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>New customer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="renewPolicyButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Renew Policy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seeCustomerListButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>See Customer List</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="customerBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Customer Actions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewPolicyDetailsButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View Policy Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Reports</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createInsuranceReportButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Insurance Report</string>
     </property>
    </widget>
   </widget>
   <zorder>deskBox</zorder>
   <zorder>titleLabel</zorder>
   <zorder>loadOptionsLabel</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder></zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>customerBox</zorder>
   <zorder>adminBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDeveloperInfo"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuReports">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Reports</string>
     </property>
     <addaction name="actionRevenueReport"/>
     <addaction name="actionInsuranceDistReport"/>
    </widget>
    <widget class="QMenu" name="menuEmployeeActions">
     <property name="title">
      <string>Actions on Employees</string>
     </property>
     <addaction name="actionAddEmployee"/>
     <addaction name="actionEditEmployee"/>
     <addaction name="actionSeeAllEmployees"/>
    </widget>
    <addaction name="actionCreateAccount"/>
    <addaction name="menuEmployeeActions"/>
    <addaction name="menuReports"/>
    <addaction name="actionRenewPolicy"/>
    <addaction name="actionCustomerList"/>
    <addaction name="actionNewCustomer"/>
    <addaction name="actionCreateInsurancesReport"/>
    <addaction name="actionViewPolicyDetails"/>
    <addaction name="actionLogout"/>
    <addaction name="actionChangePassword"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDeveloperInfo">
   <property name="text">
    <string>Developer Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCreateAccount">
   <property name="text">
    <string>Create Account(desk only)</string>
   </property>
  </action>
  <action name="actionRevenueReport">
   <property name="text">
    <string>Revenue Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="actionInsuranceDistReport">
   <property name="text">
    <string>Insurance Dist. Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="actionSeeAllEmployees">
   <property name="text">
    <string>See All Employees</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAddEmployee">
   <property name="text">
    <string>Add Employee</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionEditEmployee">
   <property name="text">
    <string>Edit Employee</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionNewCustomer">
   <property name="text">
    <string>New Customer</string>
   </property>
  </action>
  <action name="actionRenewPolicy">
   <property name="text">
    <string>Renew Policy</string>
   </property>
  </action>
  <action name="actionCustomerList">
   <property name="text">
    <string>CustomerList</string>
   </property>
  </action>
  <action name="actionCreateInsurancesReport">
   <property name="text">
    <string>Create Insurances Report</string>
   </property>
  </action>
  <action name="actionViewPolicyDetails">
   <property name="text">
    <string>View Policy Details</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
